rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.vectors/60_knn_search_filter_alias.yml

```diff
@@ -0,0 +1,282 @@
+setup:
```

Comment:
It'd be good to add another test where the alias filter requires a query rewrite. That way we can check the alias filter is rewritten before being used in kNN search. You can see `KnnSearchSingleNodeTests#testRewrite` for an example using terms lookup queries.

Reply:
As we discussed I attempted to use a terms lookup query as a filtered alias, but ran into #54550. Is there another type of filter that would guarantee a rewrite fails if not rewritten? I did add a single node test and traced through to ensure that we attempt to rewrite a filtered alias always, but obviously having a test is better.